lib/web_ui/test/text_editing_test.dart

```diff
@@ -462,6 +479,11 @@ void testMain() {
       const MethodCall show = MethodCall('TextInput.show');
       sendFrameworkMessage(codec.encodeMethodCall(show));
 
+      final MethodCall setSizeAndTransform =
```

Comment:
The "setSizeAndTransform" message has to be there since on some platforms (e.g. Desktop Safari) we don't put the input element on DOM until we get its correct dimensions from the framework.